compute/cloud-client/src/main/java/compute/disks/storagepool/CreateHyperdiskStoragePool.java

```diff
@@ -32,7 +31,7 @@ public static void main(String[] args)
     // Project ID or project number of the Google Cloud project you want to use.
     String projectId = "YOUR_PROJECT_ID";
     // Name of the zone in which you want to create the storagePool.
-    String zone = "europe-central2-b";
+    String zone = "us-central1-a";
     // Name of the storagePool you want to create.
     String storagePoolName = "YOUR_STORAGE_POOL_NAME";
     // The type of disk you want to create. This value uses the following format:
```

Comment:
The initialization should setup the code correctly instead of using the comment. Delete the second declaration of `storagePoolType` after applying the change below:
```suggestion
    // storage types can be "hyperdisk-throughput" or "hyperdisk-balanced"
    String storagePoolType = String.format( "projects/%s/zones/%s/storagePoolTypes/hyperdisk-balanced",
            projectId, zone);
```

Reply:
Thank you! Fixed